pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1159,46 +1172,20 @@ var _ = Describe("ProjectService", func() {
 			})
 
 			It("returns a Probe as expected", func() {
-				result, err := projectService.healthcheck()
+				result, err := projectService.LivenessProbe()
 				Expect(err).NotTo(HaveOccurred())
-				Expect(result).To(Equal(&v1.Probe{
+				Expect(cmp.Diff(result, &v1.Probe{
 					Handler: v1.Handler{
 						Exec: &v1.ExecAction{
-							Command: healthcheck.Test[1:],
+							Command: config.DefaultLivenessProbeCommand,
 						},
 					},
 					TimeoutSeconds:      10,
-					PeriodSeconds:       10,
-					InitialDelaySeconds: 10,
+					PeriodSeconds:       60,
```

Comment:
because they are not being read from the compose side yet, so these are the default for now, they will break once compose is read into extensions rather than using defaults